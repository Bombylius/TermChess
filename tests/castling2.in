a2 a4 a7 a6 a1 a3 a6 a5 a3 h3 b7 b6 h3 h4 .q 
