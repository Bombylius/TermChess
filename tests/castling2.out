1103806595072
16842752
282574488338432
16777216
72618344968093696
33685504
140189888446464
rnbqkbnr/2pppppp/1p6/p7/P6R/8/1PPPPPPP/1NBQKBNR b Kkq - 1 4
0
